source/support.html.erb

```diff
@@ -0,0 +1,64 @@
+---
+title: Help and support - GovWifi
+---
+<div class="container">
+  <div class="grid-row">
+    <div class="column-one-third">
+      <a href="#">A sidenav link</a>
```

Comment:
This is for a different sidebar right? 
Are you leaving it named like this for now?

Reply:
Yeah that was just a placeholder waiting for yours, which is now in.